src/parser/check_create_doc.c

```diff
@@ -0,0 +1,81 @@
+#include "../../cub3d.h"
+
+
+int  check_textures(char *str, int *err, char *orientation, int *check)
+{
+    int fd;
+
+    fd = 0;
+    move_to_space(&str);
+    if (!ft_strncmp(str, orientation, 2) && is_space(str[2]))
+    {
+        str += 2;
+        move_to_space(&str);
+        fd = open(str, O_RDONLY);
+        if (fd < 0)
+            return((*err = -1), fd);
+        *check = 1;
+    }
+    else
+        err ++;
+    return(fd);
+}
+
+
+int    try_textures(char   *str_doc, int *err_doc, t_textures *doc_texture)
```

Comment:
tirar error si nos ponen NO SO WE o EA 2 veces

Reply:
hecho aun q me da un being freed was not allocated (lo tengo que mirar)

Reply:
hecho

Reply:
excepto si lo ponen al final que lo contaria como mapa y el error lo gestionare con el checker del mapa